config.json

```diff
@@ -1727,44 +1635,44 @@
   ],
   "key_features": [
     {
-      "icon": "fun",
       "title": "Developer happiness",
-      "content": "Ruby puts developer-happiness first. It has an elegant syntax that is natural to read and write."
+      "content": "Ruby puts developer-happiness first. It has an elegant syntax that is natural to read and write.",
+      "icon": "fun"
     },
     {
-      "icon": "extensible",
       "title": "Metaprogramming magic",
-      "content": "Ruby code can write and invoke Ruby code."
+      "content": "Ruby code can write and invoke Ruby code.",
+      "icon": "extensible"
     },
     {
-      "icon": "garbage-collected",
       "title": "Garbage Collection",
-      "content": "Garbage collection done via mark and sweep, stays out of your way."
+      "content": "Garbage collection done via mark and sweep, stays out of your way.",
+      "icon": "garbage-collected"
     },
     {
-      "icon": "powerful",
       "title": "Large standard library",
-      "content": "Ruby's standard library provides a wealth of classes utilities for common tasks."
+      "content": "Ruby's standard library provides a wealth of classes utilities for common tasks.",
+      "icon": "powerful"
     },
     {
-      "icon": "tooling",
       "title": "Flexible package manager",
-      "content": "Packages (gems) can be centrally managed, but can also include custom or private gem stores."
+      "content": "Packages (gems) can be centrally managed, but can also include custom or private gem stores.",
+      "icon": "tooling"
     },
     {
-      "icon": "dynamically-typed",
       "title": "Strong, dynamic typing",
-      "content": "Ruby is strong and dynamically typed and supports 'Duck Typing'. Everything in Ruby is an object"
+      "content": "Ruby is strong and dynamically typed and supports 'Duck Typing'. Everything in Ruby is an object",
+      "icon": "dynamically-typed"
     }
   ],
   "tags": [
+    "execution_mode/interpreted",
     "paradigm/functional",
     "paradigm/object_oriented",
     "platform/linux",
-    "platform/windows",
     "platform/mac",
+    "platform/windows",
```

Comment:
You probably don't want to include this myriad of changes to this file in this PR. Let me know if you need help undoing them.

Reply:
@iHiD I know force pushing messed up the original PR I opened. If I undo these and then fixup/rebase and `--force-with-lease` will that be a problem now the the PR is open? If so I'll need some guidance on how best to commit the changes!

Reply:
Feel free to do a rebase and force push on this branch.